pkgs/top-level/aliases.nix

```
@@ -1195,6 +1195,11 @@ mapAliases ({
     Its new location is obs-studio-plugins.wlrobs.
   '';
 
+  # Added 2022-01-08
+  multimc = throw ''
+    multimc has been replaced with a fork. mutltimc is now polymc.
+  '';
```

Why the hell a _multiline_ **one-line** string here? This expression distunes completely from the style of this file.

```suggestion
  multimc = throw "multimc was removed from Nixpkgs; use polymc instead"; # Added 2022-01-08
```

---

fixed